### PR TITLE
Btree range

### PR DESCRIPTION
**Pull request by RicoGit**
fluence< put 1 v1
[fluence] 1 and v1 was written to 2 nodes
[fluence] Success.
fluence< put 2 v2
[fluence] 2 and v2 was written to 2 nodes
[fluence] Success.
fluence< range 1 2
[fluence] Reading key=1 from 192.168.1.55:11023
[fluence] Result: [(1,v1),(2,v2)]
fluence< range 1 3
[fluence] Reading key=1 from 192.168.1.55:11023
[fluence] Result: [(1,v1),(2,v2)]
fluence< range 1 9
[fluence] Reading key=1 from 192.168.1.55:11023
[fluence] Result: [(1,v1),(2,v2)]
fluence< put 3 v3
[fluence] 3 and v3 was written to 2 nodes
[fluence] Success.
fluence< put 4 v4
[fluence] 4 and v4 was written to 2 nodes
[fluence] Success.
fluence< put 5 v5
[fluence] 5 and v5 was written to 2 nodes
[fluence] Success.
fluence< range 1 9
[fluence] Reading key=1 from 192.168.1.55:11023
[fluence] Result: [(1,v1),(2,v2),(3,v3),(4,v4),(5,v5)]
fluence< get 2
[fluence] Reading key=2 from 192.168.1.55:11023
[fluence] Result: "v2"
fluence< range 0 3
[fluence] Reading key=0 from 192.168.1.55:11023
[fluence] Result: [(1,v1),(2,v2),(3,v3)]
fluence< range -1 3
[fluence] Reading key=-1 from 192.168.1.55:11023
[fluence] Result: [(1,v1),(2,v2),(3,v3)]
fluence< range 1 1
[fluence] Reading key=1 from 192.168.1.55:11023
[fluence] Result: [(1,v1)]
fluence< range -1 -1 
[fluence] Reading key=-1 from 192.168.1.55:11023
[fluence] Result: []

### CODE REVIEW

File: client/cli-app/src/main/scala/fluence/client/cli/ClientCliApp.scala [hunk 1]
```diff
@@ -30,6 +30,7 @@ import fluence.crypto.{ FileKeyStorage, SignAlgo }
 import fluence.transport.grpc.client.GrpcClient
 import monix.eval.Task
 import monix.execution.Scheduler.Implicits.global
+import monix.reactive.Observable
```
Comment (DieMyst) [inline client/cli-app/src/main/scala/fluence/client/cli/ClientCliApp.scala]: unused

File: client/cli/jvm/src/main/scala/fluence/client/cli/Cli.scala [hunk 2]
```diff
@@ -87,6 +99,14 @@ object Cli extends slogging.LazyLogging {
           } yield true
           t.toIO
 
+        case Some(CliOp.Range(from, to)) ⇒
+          val t = for {
+            res ← ds.range(from, to).toListL
+            printRes = res.mkString("[", ",", "]")
```
Comment (DieMyst) [inline client/cli/jvm/src/main/scala/fluence/client/cli/Cli.scala]: this should be in logger lazy scope

File: client/cli-app/src/main/scala/fluence/client/cli/ClientCliApp.scala (hunk 1 above)
Comment (RicoGit) [reply client/cli-app/src/main/scala/fluence/client/cli/ClientCliApp.scala]: thanks

File: client/cli/jvm/src/main/scala/fluence/client/cli/Cli.scala (hunk 2 above)
Comment (RicoGit) [reply client/cli/jvm/src/main/scala/fluence/client/cli/Cli.scala]: ok

**Pull request merged**